ui-participant/src/landing/registration/PortalRegistrationRouter.tsx

```diff
@@ -10,7 +10,7 @@ import RegistrationUnauthed from './RegistrationUnauthed'
 
 
 export type RegistrationContextT = {
-  preRegSurvey: Survey | null,
+  preRegSurvey?: Survey,
```

Comment:
This allows for `undefined` as well as `null`, right? As a reader, this does express something different to me because I see `undefined` as uninitialized and `null` as initialized but absent. That said, I'm not necessarily objecting to this change; I haven't looked to see if that's even an important distinction in this particular case.

Reply:
I think that just allows undefined, not null.  I defer to Nick on this.  I tend to like using <Object> | null for useState constructions because, like you said, it's not that it's not-thought-of, it's that it's explicitly empty